app/styles/signup.css

```diff
@@ -5,71 +5,67 @@
   padding: 1em 0;
 }
 
-.form {
-  width: 60vw;
-  max-width: 40rem;
-  min-width: 300px;
-  margin: 0 auto;
-  padding: 1em 1.5em;
-  border: 1px solid #f4f4f4;
-  border-radius: 2%;
-  box-shadow: 0 0 15px -7px rgba(0, 0, 0, 0.65);
-}
-
-.field {
-  width: 100%;
-  margin-bottom: 1em;
+.user-details {
   display: flex;
   flex-direction: column;
   justify-content: center;
-  align-items: flex-start;
+  align-content: center;
+  height: 80vh;
```

Comment:
Note: `vh` property doesn't work properly in mobile browsers.
Solution: https://css-tricks.com/the-trick-to-viewport-units-on-mobile/#:~:text=%2F%2F%20First%20we%20get%20the,documentElement.